gateway/api_definition.go

```diff
@@ -399,7 +399,7 @@ type nestedApiDefinitionList struct {
 }
 
 type nestedApiDefinition struct {
-	*apidef.APIDefinition `json:"api_definition,inline"`
+	*apidef.APIDefinition `json:"api_definition"`
```

Comment:
Please don’t remove it because it is the same structure with dashboard APIDefinition. Soon, we should use a common structure to prevent such changes.

Reply:
https://github.com/golang/go/issues/6213 - `,inline` is not implemented on go stdlib json package, so this changes nothing. The value stayed embedded and read from `api_definition` field, as defined in the tag.

Reply:
This is not a topic for this pr.

Reply:
It's like modifying a comment, it has no effect on the PR, the json package does not implement this. Do you have an idea of a test case that could fail?

Reply:
See it raises unrelated discussion to the actual issue. That’s why.

Reply:
The boyscout rule is the only way that this will realistically get merged, along with changes in the relevant code area. My first comment answers any possible concern you may have in removing `,inline`, I'd appreciate if you just click resolve conversation in this case, there isn't a conversation we can have about this, "this isn't a topic for this pr" also isn't helpful.

Reply:
If a change is made, it should be made in one ticket and all occurrences. This will create inconsistency in the code. The following ones should also be moved to one place and used in all places. Also, you are saying, removing it will not affect anything.

- https://github.com/TykTechnologies/tyk-analytics/blob/master/dashboard/model_api_definition.go#L57-L58
- https://github.com/TykTechnologies/tyk-sink/blob/ed4c1b9c0349bbe0e968ec15a53184132bce47fb/storage/store/store.go#L32-L33